### DISCUSSION TITLE
A (crowdsourced?) list of working games?

### DISCUSSION THREAD

**Discussion by Evengard** [InoriRus/Kyty#40] [Ideas]
Shouldn't we somehow crowdsource a some sort of database of games this emulator is able to run?
Maybe the easy way would be to at least create a "Category" here, in GitHub discussions, named "Games", and create here one discussion per game, updating the first post when new info is available.

**Comment by SoffiMeower** [comment]
Hi there! I made a spreadsheet for us to upload tested games! Right now it only has Amplitude and Horizon Forbidden West, but anyone can edit the sheet! 
https://docs.google.com/spreadsheets/d/1NSf_N57Qffy5CAr2QYpWi0jOMOHJjhXB9JsYNqPqGHk/edit

**Comment by K0bin** [comment]
> Shouldn't we somehow crowdsource a some sort of database of games this emulator is able to run?

That list would almost be empty. Give it time. A PS4/5 emulator is a HUGE project. It's not gonna run AAA games after one person worked on it for a year. That will take years.

**Comment by armaanhafizy86-wq** [comment]
[](https://docs.google.com/spreadsheets/d/1q8_kLcIze6gx5yQqv4x2LHIoU8bXe002QNzXb-o6r9M/edit?gid=0#gid=0) But my version was used with ai…

**Comment by armaanhafizy86-wq** [comment]
[https://docs.google.com/spreadsheets/d/1q8_kLcIze6gx5yQqv4x2LHIoU8bXe002QNzXb-o6r9M/edit?gid=0#gid=0](url)
